docs/docs/advanced-topics/6-using-cli.md

```diff
@@ -124,67 +124,6 @@ typeorm subscriber:create path-to-subscriber-dir/subscriber
 
 Learn more about [Subscribers](./4-listeners-and-subscribers.md).
 
-## Create a new migration
-
-You can create a new migration using CLI:
-
-```shell
-typeorm migration:create path-to-migrations-dir/migrationName
-```
-
-Learn more about [Migrations](./1-migrations.md).
```

Comment:
this type of information is still useful under the cli docs.

Reply:
I don't think it is. One should learn and understand how migrations works from _migrations_ doc. If anything this page should only list all cli commands and reference to other sections in the docs (which was my longer term plan...). I just restored it in that form, please have a look.

Worth to note, that current CLI docs are outdated/not complete and the information how to run things via cli is spread across the whole docs...